asset/css/markbind.css

```diff
@@ -121,3 +121,32 @@ footer {
 #flex-div {
     flex: 1;
 }
+
+/* Spinner */
+
+#spinner-overlay {
+    position: fixed;
+    z-index: 100;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background: white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#spinner {
+    border: 16px solid lightgray;
+    border-top: 16px solid darkgray;
+    border-radius: 50%;
+    width: 120px;
+    height: 120px;
+    animation: spin 2s linear infinite;
+}
+
+@keyframes spin {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+}
```

Comment:
As stated earlier, we can use the Bootstrap's spinner, so these should no longer be necessary.